news-aggregation/services/NewsFetchService.ts

```diff
@@ -0,0 +1,85 @@
+import {
```

Comment:
This file is not needed I guess, since we are handling the logic in the scheduled job using the FetchNewsProcessor.

Reply:
I have kept this file since the `FetchNewsProcessor ` is not working on my machine. So that I can manually fetch the news. (for development)